### PR TITLE
Sanitize some of content in the repo

### PR DESCRIPTION
**Pull request by mgianatagh**

### CODE REVIEW

**Review by markxnelson — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -270,10 +271,10 @@ pipeline {
                     cd ${GO_REPO_PATH}/verrazzano/operator
                     git config --global credential.helper "!f() { echo username=\\$DOCKER_CREDS_USR; echo password=\\$DOCKER_CREDS_PSW; }; f"
                     git config --global user.name $DOCKER_CREDS_USR
-                    git config --global user.email "70212020+verrazzanobot@users.noreply.github.com"
+                    git config --global user.email "${DOCKER_EMAIL}"
                     git checkout -b ${env.BRANCH_NAME}
                     git add deploy/operator.yaml
-                    git commit -m "[verrazzanobot] Update verrazzano-platform-operator image version to ${DOCKER_IMAGE_TAG} in operator.yaml"
+                    git commit -m "[verrazzano] Update verrazzano-platform-operator image version to ${DOCKER_IMAGE_TAG} in operator.yaml"
```
Comment (bobdonat) [inline Jenkinsfile:277]: I think you should check with Mike Gilbode on this since he suggested verrazzanobot

Comment (mgianatagh) [reply Jenkinsfile:277]: I raised this exact question on the initial review, and Mark identified we should not be checking in a username.  If I recall correctly, @gilbode wanted this to be some static text.   Is `verrazzano` ok?

Comment (gilbode) [reply Jenkinsfile:277]: Verrazzano is fine

File: operator/config/samples/install-olcne.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ spec:
           value: "2"
         - name: controller.service.externalIPs
           valueList:
-            - 152.67.140.131
+            - 11.22.33.44
```
Comment (gilbode) [inline operator/config/samples/install-olcne.yaml:26]: What are these IP addresses?

Comment (mgianatagh) [reply operator/config/samples/install-olcne.yaml:26]: For OLCNE install, the load balancers must be pre-allocated before the install.  These are the external IP addresses of those load balancers.

**Pull request merged**